images/image-builder/run.sh

```diff
@@ -5,6 +5,19 @@ export registry_path="${2}"
 
 source /usr/local/bin/dind.sh
 
+# Configuring docker-credential-helper-ecr
+cat > ~/.docker/config.json << EOF
+{
+"credsStore": "ecr-login"
+}
+{
+	"credHelpers": {
+		"public.ecr.aws": "ecr-login",
+		"<aws_account_id>.dkr.ecr.<region>.amazonaws.com":"ecr-login"
```

Comment:
I wonder if this is required based on https://github.com/awslabs/amazon-ecr-credential-helper#docker

Reply:
I removed the credHelpers. It is not needed for pull and push for docker images.